source/extensions/omni.isaac.lab/test/utils/test_math.py

```diff
@@ -376,6 +376,24 @@ def iter_old_quat_rotate_inverse(q: torch.Tensor, v: torch.Tensor) -> torch.Tens
                 iter_old_quat_rotate_inverse(q_rand, v_rand),
             )
 
+    def test_depth_perspective_conversion(self):
```

Comment:
Please always add a docstring for a test so the description also is visible when you run the test.